### PR TITLE
texlive: exclude dvisvgm

### PR DESCRIPTION
**Pull request by mojca**
The dvisvgm binary is available as a standalone port which depends on ghostscript.

Removing it from the TeX Live package has a number of benefits:
* No need to revbump `texlive-bin` after each `ghoscript` update.
* No need for a (missing) conflict between `texlive-bin-extra` and `dvisvgm`.
* Users of `texlive-bin-extra` get a newer version of `dvisvgm`.

See also: #2573, #2620.

#### Description

This looks to me like the correct thing to do instead of blindly (and uselessly) revbumping `texlive-bin` after each ghoscript update (#2573).

@l2dy: I would commit this together with #2620 and only do the revbump once. It could be a bit of a bordercase to do this quickly: this commit would generally not fall under openmaintainer policy to be committed under 72 hours, but @drkp did not respond to my request for review of #2573 and both cited PRs are security issues which do fall under "your-are-allowed-to-commit-immediately" policy. Commiting this at the same time would merely spare some CPU cycles. I leave it up to others to decide. I hope that @drkp responds quickly.

@RasmusWernerLarsen

###### Type(s)

- [x] bugfix

###### Tested on

macOS 10.13.6 17G65
Xcode 9.4.1 9F2000

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
  * commits would ideally be combined with #2620
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
  * two open pull requests referenced above
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by pmetzger** [comment]
This would seem like a good idea if only because dvisvgm is of use to people who don't use TeX as well.

File: tex/texlive-bin-extra/Portfile [hunk 1]
```diff
@@ -28,12 +28,13 @@ depends_lib         port:texlive-basic
...
 
 depends_run         port:latexmk \
                     port:detex \
                     port:latexdiff \
                     port:pdfjam \
-                    port:dvipng
+                    port:dvipng \
+                    port:dvisvgm
```
Comment (l2dy) [inline tex/texlive-bin-extra/Portfile]: Should `conflicts           texlive-bin-extra` be removed from graphics/dvisvgm/Portfile then?

Comment (mojca) [reply tex/texlive-bin-extra/Portfile]: Yes, definitely, thanks a lot for noticing. I wondered why there was no conflict in `texlive-bin-extra` and it didn't complain when I tried to install `texlive-bin-extra`, but I didn't check `dvisvgm` itself.

Comment (mojca) [reply tex/texlive-bin-extra/Portfile]: I'm in the middle of working on some other ports which will take forever to build, so feel free to change this and commit it together with your other patch.

**Comment by drkp** [comment]
Merged this in 4049a91919

**Pull request closed without merging**